transforms/universal/ededup/ray/src/ededup_transform_ray.py

```diff
@@ -108,12 +118,53 @@ def get_transform_config(
         :return: dictionary of transform init params
         """
         # create hashes
-        self.filters = RayUtils.create_actors(
-            clazz=ray.remote(HashFilter),
-            params={},
-            actor_options={"num_cpus": self.params.get("hash_cpu", 0.5)},
-            n_actors=self.params.get("num_hashes", 1),
-        )
+        n_filters = self.params.get(num_hashes_key, 1)
+        self.filters = [None] * n_filters
+        for i in range(n_filters):
+            self.filters[i] = (
+                ray.remote(HashFilter)
+                .options(num_cpus=self.params.get(hash_cpu_key, 0.5))
+                .remote({"id": i, "data_access_factory": data_access_factory})
+            )
+        if self.params.get(use_snapshot_key, False):
+            # we are using snapshots. Note here that the amount of files might be different
```

Comment:
maybe this long bit of code under this if could go in a _load_snapshots() method

Reply:
done